cli/dcoscli/node/main.py

```diff
@@ -473,7 +473,7 @@ def _list(json_):
     if json_:
         emitter.publish(slaves)
     else:
```

Comment:
we should verify that the field names exist before we try to print them in the table